server.py

```diff
@@ -1,16 +1,86 @@
+import json
```

Comment:
dont commit any of this server or web stuff; it is pretty half baked

Reply:
I keep trying to not but it get stuck in the PR for some reason